src/vsphere_cpi/spec/unit/cloud/vsphere/vcenter_client_spec.rb

```diff
@@ -177,9 +194,25 @@ module VSphereCloud
         end
       end
 
+
+      context 'when file manager raises "Invalid datastore path" error for the disk' do
+        it 'does not raise error' do
+          expect(task_runner).to receive(:run) do |*args, &block|
+            expect(block.call).to eq(vmdk_task)
+          end.and_raise(RuntimeError.new('Invalid datastore path some/path'))
```

Comment:
No need to explicitly to call `RuntimeError.new` here. Just do:

```ruby
and_raise(RuntimeError, 'Invalid datastore path some/path')
```

Or better yet:

```ruby
and_raise('Invalid datastore path some/path')
```

Since `RuntimeError` is the default exception class.